src/main/python/covata/delta/api/signer.py

```diff
@@ -52,51 +53,52 @@ def canonical_request(self):
         return self.__canonical_request
 
 
-class CVTSigner(AuthBase, LogMixin):
```

Comment:
revert this class, the signer should also be a generic class which should be inherited by RequestsCVTSigner.